capstone/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -30,19 +32,20 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     UserService userService = UserServiceFactory.getUserService();
 
-    String loginUrl = userService.createLoginURL("/group.html");
+    String loginUrl = userService.createLoginURL("/");
     String logoutUrl = userService.createLogoutURL("/");
```

Comment:
inline these where you are creating LoginResponse since you aren't using them elsewhere